hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/utils/AsyncRFAAppender.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.utils;
+
+import java.io.IOException;
+
+import org.apache.log4j.AsyncAppender;
+import org.apache.log4j.PatternLayout;
+import org.apache.log4j.RollingFileAppender;
+import org.apache.log4j.spi.LoggingEvent;
+
+/**
+ * The AsyncRFAAppender shall take the required parameters for supplying
+ * RollingFileAppender to AsyncAppender.
+ */
+public class AsyncRFAAppender extends AsyncAppender {
+
+  private String maxFileSize = String.valueOf(10 * 1024 * 1024);
+
+  private int maxBackupIndex = 1;
+
+  private String fileName = null;
+
+  private String conversionPattern = null;
+
+  private final Object conversionPatternLock = new Object();
```

Comment:
`conversionPatternLock` seems to be unnecessary after making all access to `conversionPattern` sync on the appender itself.

Reply:
Thank you, made the changes!